src/controls/editor-mode/components/unique-option-controls.js

```diff
@@ -6,56 +6,112 @@ import { makeStyles } from '@eqworks/lumen-labs'
 import { useStoreActions, useStoreState } from '../../../store'
 import typeInfo from '../../../constants/type-info'
 import types from '../../../constants/types'
-import { renderCheckbox } from '../../shared/util'
+import { renderCheckbox, renderToggle, renderMultiSelect, renderRadioSelect } from '../../shared/util'
+import { tableUniqueOptions, borderType, headerColor } from '../../../constants/table-widget'
 
 
 const useStyles = ({ gridCols }) => makeStyles({
-  uniqueOptionsContainer : {
+  uniqueOptionsBooleanContainer : {
     display: 'grid',
     gridTemplateColumns: `repeat(${gridCols}, 1fr)`,
     alignItems: 'center',
     columnGap: '0.75rem',
+
+    '& .table-bar__select-options': {
+      gridColumn: 'span 2',
+
+      '& .radio-select__root-container': {
+        marginBottom: '0.313rem',
+      },
+    },
   },
 })
 
 const UniqueOptionControls = ({ type }) => {
   const userUpdate = useStoreActions((actions) => actions.userUpdate)
   const uniqueOptions = useStoreState((state) => state.uniqueOptions)
+  const renderableValueKeys = useStoreState((state) => state.renderableValueKeys)
 
   const gridCols = useMemo(() => {
     const options = Object.keys(typeInfo[type]?.uniqueOptions || {}).length
 
-    if ([2, 4].includes(options)) {
+    if ([2, 4].includes(options) || types.TABLE === type) {
       return 2
     }
     return 3
   }, [type])
 
+  const tableData = useMemo(() => {
+    return renderableValueKeys.map(({ title }) => title)
+  }, [renderableValueKeys])
+
+  const getTableRadioSelectionData = (type) => {
+    if (tableUniqueOptions[type] === 'borderType') return borderType
+    if (tableUniqueOptions[type] === 'headerColor') return headerColor
+  }
+
   const classes = useStyles({ gridCols })
 
   return (
-    <div className={classes.uniqueOptionsContainer}>
-      {Object.entries(typeInfo[type]?.uniqueOptions || {})
-        .map(([k, { name, type: _type }], i) => {
-          switch (_type) {
-            case Boolean: // TODO support types other than bool
-              return (
-                <div key={i}>
-                  {renderCheckbox(
-                    name,
-                    uniqueOptions[k],
-                    v => userUpdate({ uniqueOptions: { [k]: v } }),
-                    false,
-                    i
-                  )}
-                </div>
-              )
-            default:
-              return null
-          }
-        })
-      }
-    </div>
+    <>
+      <div className={classes.uniqueOptionsBooleanContainer}>
```

Comment:
This breaks with the styling for the other widgets. Please check the styling for Display options (specifically the space between the rows of different elements) for the bar widget stories and add styling to match the section there.

The lower part grid for the outlined area should use all the space in the row, each radio should have the same space.
<img width="355" alt="Screen Shot 2023-02-08 at 4 53 23 PM" src="https://user-images.githubusercontent.com/41120953/217660015-119074f0-0ba7-4298-97c3-e9fffb28b053.png">